src/main/java/com/metamx/cache/CaffeineCache.java

```diff
@@ -45,17 +47,29 @@
 
   public static CaffeineCache create(final CaffeineCacheConfig config)
   {
+    return create(config, null);
+  }
+
+  // Used in testing
+  public static CaffeineCache create(final CaffeineCacheConfig config, @Nullable final Executor executor)
+  {
     Caffeine<Object, Object> builder = Caffeine.newBuilder().recordStats();
     if (config.getExpiration() >= 0) {
-      builder = builder
+      builder
           .expireAfterAccess(config.getExpiration(), TimeUnit.MILLISECONDS);
```

Comment:
is there a reason to have an expiration time for local cache? I could see expiration times adding more overhead to the cache and not adding any benefit.